src/stories/views/components/forms/message.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="flex flex-row items-center justify-between w-full">
+        <div class="flex items-center gap-x-2 md:gap-x-3">
+            {{#if _success}}
+                {{> components/forms/success_icon}}
+                <span class="font-bold text-green-800">{{_messageText}}</span>
+                </div>    
+            {{else}}
+                {{> components/forms/error_icon}}
+                <span class="font-bold text-red-700">{{_messageText}}</span>
+                </div>
```

Comment:
das das div hier und in Zeile 6 jeweils geschlossen wird finde ich irgendwie unschön.
mach doch lieber noch drunter nochmal ein unless _success wo du den button reinpackst. Oder?